.travis.yml

```diff
@@ -31,6 +31,8 @@ before_install:
   # - brew outdated carthage || brew upgrade carthage
   - Carthage update --platform iOS  --cache-builds
   - gem i slather
+  # Install build output formatting tool
+  - gem install xcpretty
```

Comment:
This is already done in another PR so you can revert it from here

Reply:
fixed